### ISSUE TITLE
[Feature Request] Field for X-Original-To

### ISSUE THREAD

**Issue by dartrax** [barbushin/php-imap#604]
**Is your feature request related to a problem? Please describe.**
I've seen mails where the recipients address is not in the "To" or any other field of the email header, only in the "X-Original-To". As an example, the notification mails I receive from github. May be it occurs when a mail was sent only using bcc?

**Describe the solution you'd like**
I'd like to have the X-Original-To available like this:
`$email->x_original_to`

**Describe alternatives you've considered**
May be I've overseen something, but the only way I could find was to parse the headersRaw like so:

```
if (preg_match('/(X-Original-To:\s)(\S*@\S*)/', $email->headersRaw, $matches))
    $xoriginalto = $matches[2];
```

**Comment by Sebi94nbg** [comment]
I've added this header property. It's currently only on the `develop` branch.

@dartrax Can you please check, if it's properly working for you as I don't have any example to test?

```php
$email->xOriginalTo
```

**Comment by dartrax** [comment]
Hi @Sebi94nbg,
thanks for implementing that! I've tested it today with two different mails and so far it worked!
I'll use it in my personal mail filter/forwarding script from now on so it'll see nearly every mail I get and I'll report when I encounter any issues.

**Comment by Sebi94nbg** [comment]
Great. Thanks for testing and your feedback! :)

I'll hopefully find this week the left time to fix the current pipeline issues and then I'll publish this change in a new release. :)

**Comment by Sebi94nbg** [comment]
This change is included in the release [4.3.0](https://github.com/barbushin/php-imap/releases/tag/4.3.0) and newer.
